great_expectations/core/factory/suite_factory.py

```diff
@@ -94,3 +94,9 @@ def get(self, name: str) -> ExpectationSuite:
         if self._include_rendered_content:
             suite.render()
         return suite
+
+    @public_api
+    @override
+    def all(self) -> Iterable[ExpectationSuite]:
+        """Get all ExpectationSuites."""
+        return []
```

Comment:
Should this be `return self._store.get_all()`?

Reply:
Hah sorry, this should have been marked as a draft! It certainly should be, but at the time I put this up those weren't yet implemented everywhere!